docs/src/inflation.md

```diff
@@ -0,0 +1,42 @@
+# Inflation 
+Inflation is a approach that slows ensemble collapse in Kalman methods.
+Two distinct forms of inflation are implemented, in which perturbations are applied immediately following the standard Kalman update. 
+Multiplicative inflation expands ensemble members away from their mean in a
+deterministic manner, whereas additive inflation hinges on the addition of stochastic noise to ensemble members.
+For both implementations, a scaling factor ``s`` is included to extend functionality to cases with mini-batching. 
+The scaling factor ``s`` multiplies the artificial time step ``\Delta t`` in the inflation equations to account for sampling error. For mini-batching, the scaling factor should be:
+```math
+    s = \frac{|B|}{|C|}
+```
+where `` |B| `` is the mini-batch size and `` |C| `` is the full dataset size.
+
+## Multiplicative Inflation 
+Multiplicative inflation effectively scales parameter vectors in parameter space, such that the perturbed
+ensemble remains in the linear span of the original ensemble. The implemented update equation follows
+[Huang et al, 2022](https://arxiv.org/abs/2204.04386) eqn. 41:
+
+```math
+\begin{aligned}
+    m_{n+1} = m_{n} ; \qquad u^{j}_{n + 1} = m_{n+1} + \sqrt{\frac{1}{1 - s \Delta{t}}} \left(u^{j}_{n} - m_{n} \right) \qquad (1)
+\end{aligned}
+```
+where ``m`` is the ensemble average. In this way,
+the variance across parameter vector magnitudes is increased by a factor of ``\frac{1}{1 - s \Delta{t}}``, while the mean remains fixed.
+
+Multiplicative inflation can be used by flagging the `update_ensemble!` method as follows:
+```julia
+    EKP.update_ensemble!(ekiobj, g_ens; multiplicative_inflation = true, s = 1.0)
+```
+
+## Additive Inflation 
+Additive inflation is implemented by systematically adding stochastic perturbations to the parameter ensemble in the form of gaussian noise. Additive inflation breaks the linear subspace property, meaning the parameter ensemble can evolve outside of the span of the initial ensemble. In additive inflation, the ensemble is perturbed in the following manner after the standard Kalman update:
+
+```math
+     u_{n+1} = u_n + \zeta_{n} \qquad (2) \\
+    \zeta_{n} \sim N(0, \frac{s \Delta{t} }{1 - s \Delta{t}} C_n) \qquad (3)
+```
+This increases the variance across vector components by `` \frac{s \Delta{t} }{1 - s \Delta{t}} ``, while the mean remains fixed.
```

Comment:
Be careful when saying "increase", as above we **multiply** the variance by `1/(1-sDelt)` whereas here we **add** `sDelt/(1-sDelt)` to the variance.
Actually these give a new ensemble with the same variance right? Adding noise to each particle gives new variance scaled by `1 + sDelt/(1-sDelt) = (1-sDelt + sDelt) / (1-sDelt) = 1/(1-sDelt)` Might be worth clearing this up

Reply:
Oh we need C_n here